src/main/java/org/wildfly/security/ssl/Protocol.java

```diff
@@ -79,7 +80,7 @@
      * @return an enum item
      */
     public static Protocol forName(final String name) {
-        return map.get(name);
+        return map.get(name.toUpperCase(Locale.ENGLISH));
```

Comment:
Here there is theoretical chance null will come into method.
At least in class MechanismDatabase there is this marginal case handled when protocol is missing when MechanismDatabase.properties is not filled properly
```
                final Protocol prot = Protocol.forName(strings[5]);
                if (prot == null) {
                    log.warnInvalidProtocolForMechanismDatabaseEntry(strings[5], name);
                    ok = false;
                }
```

Reply:
This is not necessary - previous implementation also returns NullPointerException if the specified key is null.

Reply:
@mchoma I can update ```Protocol.forName()``` to return null when the protocol name is null.